### ISSUE TITLE
The package read messages only if pub/sub is used

### ISSUE THREAD

**Issue by cedrick-ah** [Transfa/sendhooks-engine#84]
I am not good at redis.  Trying to use this package  but was not able to get messages and the payload. When modified to use the redis [Publish/Subscribe messaging paradigm](http://en.wikipedia.org/wiki/Publish/subscribe) I was able to get messages from the client redis  client. Seeing the way the client publish in the [doc](https://sendhooks.com/docs/quickstart) inspired me.

**Comment by koladev32** [comment]
Yes, we are not using Pub/Sub because of the cons regarding the messages. With the pub/sub model of Redis, we lose the message if it is not delivered. So we are using Redis streams https://redis.io/docs/latest/develop/data-types/streams/

Also my bad, the docs on the website are not updated. I will do it in a few hours. Check this code example to see how to send the data. 

https://github.com/Transfa/sendhooks-engine/blob/main/example-project/api/app.py
